File: testing/highlight-samples/sample.cs
// This is a comment
using System;

namespace App {
    public class User {
        public string Name { get; set; } = "hello world";
        public int Age { get; set; } = 42;
        public bool Active = true;

        public string Greet() {
            return $"Hi, {Name}!";
        }

        /* Block comment */
        char initial = 'A';
    }
}

File: testing/highlight-samples/sample.cpp
// This is a comment
#include <string>
#include <iostream>

namespace app {

class User {
public:
    std::string name = "hello world";
    int age = 42;
    bool active = true;

    std::string greet() {
        return "Hi, " + name + "!";
    }
};

/* Block comment */
auto raw = R"(raw string)";

} // namespace app
